src/OSTSdk.php

```diff
@@ -26,10 +26,10 @@ class OSTSdk
     public function __construct(array $params)
     {
         // Extract API major version
-        $apiEndpointVersion = explode('/', $params['apiBaseUrl']);
-        if (empty($apiEndpointVersion[3])) {
+        $apiEndpointVersion = explode('/', $params['apiBaseUrl'])[3];
```

Comment:
If the array key for whatever reason does not exists i think it will throw warning/notice?